radiation/python/radlw/setcoef.py

```diff
@@ -271,12 +272,12 @@ def setcoef(pavel, tavel, tz, stemp, h2ovmr, colamt, coldry, colbrd,
 outdict = dict()
 
 for var in invars:
-    tmp = serializer.read(var, savepoints[0])
+    tmp = serializer.read(var, savepoints[10])
```

Comment:
I think reading serialized savepoints by name would help reduce the maintenance load needed in cases like this.

Reply:
Done